build.sbt

```diff
@@ -1,27 +1,71 @@
-name := "play-mailer"
-
-organization := "net.kaliber"
-
-scalaVersion := "2.11.8"
+//name := "play-mailer"
 
 resolvers ++= Seq(
   "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
   "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases" /* specs2-core depends on scalaz-stream */
 )
 
-val playVersion = "2.5.0"
+lazy val commonSettings = Seq(
+  organization := "net.kaliber",
+  scalaVersion := "2.11.8"
+)
+
+lazy val core = (project in file("core"))
+  .settings(
+    name := "play-mailer-core"
```

Comment:
Yeah, that was also my issue. Since its in the play-mailer repo I couldnt think of a better name, but I like "scala-mailer"